inventory/src/androidTest/java/org/flyve/inventory/SimcardsTest.java

```diff
@@ -1,42 +1,45 @@
 /**
- *  LICENSE
- * 
- *  This file is part of Flyve MDM Inventory Library for Android.
+ * LICENSE
+ * <p>
```

Comment:
Hi, @Ivans51 
There are extra HTML code in the License header.
And add also your name and email in the headers.